kratos/processes/vtk_output_process.cpp

```diff
@@ -55,29 +55,29 @@ int VtkOutputProcess::Check()
 
 
 
-void VtkOutputProcess::CreateMapFromKratosIdToVTKId(ModelPart &model_part)
+void VtkOutputProcess::CreateMapFromKratosIdToVTKId(ModelPart &rModelPart)
 {
     int vtk_id = 0;
 
-    for (ModelPart::NodeIterator node_i = model_part.NodesBegin(); node_i != model_part.NodesEnd(); ++node_i)
+    for (ModelPart::NodeIterator node_i = rModelPart.NodesBegin(); node_i != rModelPart.NodesEnd(); ++node_i)
```

Comment:
This and the following loops can be clearer and simpler using the C++11 way (I wwill not block because of this, but the code will look simpler and nicer):

~~~cpp
for (auto& r_node : rModelPart.Nodes() )
{
         const std::size_t KratosId = r_node.Id();
         mKratosIdToVtkId[KratosId] = vtk_id;
         ++vtk_id;
     }
~~~

The same for the following loops

Reply:
As I said I am continuously changing things in here, all the time keeping it working for others to use/test it. No only this change you mentioned here, I am sure there are also others .. !